content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,224 @@ aliases:
+
+The number of unique combinations of metric name, host, and tag values submitted for a `GAUGE` metric with [this tagging scheme is 4](#counting-custom-metrics?tab=countrategauge):
+
+| Metric Name       | Tag Values                           |
+|-------------------|--------------------------------------|
+| `request.Latency` | `host:A`, `endpoint:X`, `status:200` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:200` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:400` |
+| `request.Latency` | `host:B`, `endpoint:Y`, `status:200` |
+
+Given `request.Latency` is submitted as a `DISTRIBUTION`, Datadog [stores 5 custom metrics][1] for each of the original 4 unique tag value combinations to account for server side-aggregations: `count`, `sum`, `min`, `max`, and `avg`.
+
+This results in a total of 20 custom metrics for this `DISTRIBUTION` metric:
+
+| Metric Name       | Tag Values                           | Aggregations                            |
+|-------------------|--------------------------------------|-----------------------------------------|
+| `request.Latency` | `host:A`, `endpoint:X`, `status:200` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:200` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:400` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:B`, `endpoint:Y`, `status:200` | `count`, `sum`, `min`, `max`, and `avg` |
+
+##### Distribution Metrics with Additional Percentile Aggregations
+
+[Adding additional percentile aggregations][2]: `p50`, `p75`, `p90`, `p95`, or `p99` to a distribution metric increases the number of distinct custom metrics reported. In order to support additional globally accurate percentile aggregations, a dedicated custom metric is created for a given aggregation per every potentially queryable tag value combination in your data. For instance let's say you have two hosts: `host:A` and `host:B` and you want to calculate the global `p50` (the median) of your requests latency over a given interval:
+
+| Host     | Requests Latencies measured (in ms) | p50 |
+|----------|-------------------------------------|-----|
+| `host:A` | 10,20,30,40,50,60,70,80,90,100,110  | 60  |
+| `host:B` | 10,15,20,25,30                      | 20  |
+
+The overall `p50` of latency for your request is **NOT** `avg(60,20)=40 ms`, the correct `p50` is: `median(10,10,15,20,20,25,30,30,40,50,60,70,80,90,100,110)=35 ms`
+
+For other metric types, like GAUGE and COUNT, Datadog can combine/aggregate the most granular data to obtain globally accurate results. Percentiles can’t be recombined; therefore, they are precomputed and stored individually in a dedicated custom metric per every potentially queryable tag value combination in your data
+
+**Note**: You can control [which percentile aggregation are performed][2] over [which tag combination][3] for any DISTRIBUTION metric.
```

Comment:
This note is incorrect - people can't control which percentile aggregations are stored -- all 5 {p50,p75,p90,p95,p99} are stored. BUT they can control what tag value combinations are potentially queryable depending on their whitelist of tags